lychee-bin/src/options.rs

```diff
@@ -259,7 +259,7 @@ impl Config {
             Err(e) => {
                 return match e.kind() {
                     ErrorKind::NotFound => Ok(None),
-                    _ => Err(Error::from(e)),
+                    _ => Err(e.into()),
```

Comment:
No I deliberately avoid using Into::into().
It's hard to figure out what it is converting to.

Reply:
We are calling `.into()` inside of an `Err(...)`, the `Error::` doesn't give you any new information here and just makes the more verbose.

Reply:
How do you figure out what's the type of `E` in a `Result<T, E>`?
`Error::from(e)` clearly shows `E` is of type `Error`, while `e.into()` has no such information.